go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	k8s.io/code-generator v0.24.3
 	k8s.io/component-base v0.24.3
 	k8s.io/klog/v2 v2.70.1
-	k8s.io/kube-openapi v0.0.0-20220328201542-3ee0da9b0b42
+	k8s.io/kube-openapi v0.0.0-20220328201542-3ee0da9b0b42 // indirect
```

Comment:
The addition of the `// indirect` notation is correct.  This is distinct from the rebase to v0.11.0; this notation addition is needed even without the rebase.

The convention seems to be that direct requirements are listed in the first r`equire` block and indirect ones are listed in the second `require` block.